identity-mdoc/src/commonMain/kotlin/com/android/identity/mdoc/nfc/scanNfcMdocReader.kt

```diff
@@ -33,21 +30,39 @@ data class ScanNfcMdocReaderResult(
  *
  * This uses [scanNfcTag] to show a dialog prompting the user to tap the mdoc.
  *
+ * When a connection has been established, [onHandover] is called with the created transport as well as
```

Comment:
nit: could we wrap at 80 columns? It's friendlier to github's code review tool, and possibly other tools we may use.
...should be a setting in Android Studio.

Reply:
Yep, I'm familiar with the setting and mine is set at 120 🤣  ... so, that is to say, I agree we should wrap at N but I think 80 is way too small, 120 seems just right. Let's revisit this later as part of linting so we can enforce it...